jung-algorithms/src/main/java/edu/uci/ics/jung/layout/util/RadiusNetworkNodeAccessor.java

```diff
@@ -27,15 +27,14 @@
  * @author Tom Nelson
  * @author Joshua O'Madadhain
  */
-public class RadiusNetworkNodeAccessor<N, P> implements NetworkNodeAccessor<N, P> {
+public class RadiusNetworkNodeAccessor<N> implements NetworkNodeAccessor<N> {
 
   private static final Logger log = LoggerFactory.getLogger(RadiusNetworkNodeAccessor.class);
   protected final Graph<N> graph;
   protected double maxDistance;
-  protected PointModel<P> pointModel;
   /** Creates an instance with an effectively infinite default maximum distance. */
-  public RadiusNetworkNodeAccessor(Graph<N> graph, PointModel<P> pointModel) {
-    this(graph, pointModel, Math.sqrt(Double.MAX_VALUE - 1000));
+  public RadiusNetworkNodeAccessor(Graph<N> graph) {
```

Comment:
Maybe they should be named MinimumRadiusNodeFunction (just Nodes) and MinimumRadiusElementFunction (Nodes and Edges)

Reply:
I think `Accessor` is better than `Function` (less generic, and this isn't a Function in the `@FunctionalInterface` sense), but there may be better names still.  Maybe `NodeFinder` and `ElementFinder` for the base interfaces.

On reflection, I don't favor `MinimumRadius` as a name because that's definitionally what all of these things do (find the closest thing); this one simply happens to do a linear search.

Perhaps `LinearSearch*Finder`, and then things like `QuadTree*Finder` and `RTree*Finder`.  Thoughts?